config/config.exs

```diff
@@ -21,6 +21,10 @@ use Mix.Config
 #     config :logger, level: :info
 #
 
+config :logger, :console,
+  format: "\n$time $metadata [$level] $message",
```

Comment:
Will this conflict with the config in BM?  Will BM's format overwrite this?

Reply:
See response below -- the config from a dependency doesn't get used/imported, only the config of the actual project you're compiling/running.